src/test/java/EventTest.java

```diff
@@ -0,0 +1,23 @@
+package test.java;
+
+import main.java.duke.Event;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
```

Comment:
Same thing for this class, perhaps you could rename your test methods so that it is easier to understand?
`featureUnderTest_testScenario_expectedBehavior()`